kratos/utilities/geometry_utilities.h

```diff
@@ -46,7 +46,7 @@ class GeometryUtils
      * @param the volume of the element
      */
     static inline void CalculateGeometryData(
-        Element::GeometryType& geom,
+        const Element::GeometryType& geom,
```

Comment:
Can you change the declarations to fit the style guide?, geom -> rGeometry

Reply:
Thanks for the hint @loumalouomega ! Is it ok now?